src/components/IconButtonMenu/PIconButtonMenu.vue

```diff
@@ -32,29 +32,19 @@
   import POverflowMenu from '@/components/OverflowMenu/POverflowMenu.vue'
   import PPopOver from '@/components/PopOver/PPopOver.vue'
   import { useAttrsStylesAndClasses } from '@/compositions/attributes'
-  import { PositionMethod } from '@/types'
   import { Icon } from '@/types/icon'
   import { topRight, bottomRight, bottomLeft, topLeft } from '@/utilities/position'
 
-  const props = withDefaults(defineProps<{
+  withDefaults(defineProps<{
     icon?: Icon,
-    placement?: PositionMethod | PositionMethod[],
-    preventCloseOnClick?: boolean,
```

Comment:
Dang, I remember this being added and it making me sad. Happy to see it removed!